sdk/schemaregistry/schema-registry/package.json

```diff
@@ -19,17 +19,17 @@
     "execute:samples": "dev-tool samples run samples-dev",
     "extract-api": "tsc -p . && api-extractor run --local",
     "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
-    "integration-test:browser": "karma start --single-run",
-    "integration-test:node": "nyc mocha -r esm --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 5000000 --full-trace \"dist-esm/test/{,!(browser)/**/}*.spec.js\"",
+    "integration-test:browser": "dev-tool run test:browser",
+    "integration-test:node": "dev-tool run test:node-js-input -- --timeout 5000000 'dist-esm/test/**/*.spec.js'",
```

Comment:
Do we really even use this? This is not a comment about this PR, but there is zero difference between this and running from the TS source. ts-node will just compile the TS sources into ESMs first. It runs the exact same suite of tests as our unit-tests, so it's not really a different kind of testing.

Reply:
I'm unsure if nyc works as expected with ts files as input.

Reply:
Let's revisit this. I'll log an issue.

Reply:
@witemple-msft I think there is a confusion here. The difference here is not just running from js or ts sources. `test:node-js-input` runs `nyc` with mocha but `test:node-ts-input` runs mocha only.

@harsha-nalluru I think the names of these commands could be better here since it is not just about running js or ts. Could you consider renaming?

Reply:
@deyaaeldeen  he's suggesting using ts sources for everything instead of js and ts.
Like I said above, to do that, we'll have to verify/investigate if we're losing anything w.r.t the code coverage.

Reply:
#19735

Reply:
Please make sure to raise your voice at that issue instead.

Reply:
I've left some comments over there.